packages/core/scripts/sentry-xcode-debug-files.sh

```
@@ -17,23 +17,41 @@ set -e
 LOCAL_NODE_BINARY=${NODE_BINARY:-node}
 
+# The project root by default is one level up from the ios directory
+RN_PROJECT_ROOT="${PROJECT_DIR}/.."
+
 [ -z "$SENTRY_PROPERTIES" ] && export SENTRY_PROPERTIES=sentry.properties
+[ -z "$SENTRY_DOTENV_PATH" ] && export SENTRY_DOTENV_PATH="$RN_PROJECT_ROOT/.env.sentry-build-plugin"
 
 [ -z "$SENTRY_CLI_EXECUTABLE" ] && SENTRY_CLI_PACKAGE_PATH=$("$LOCAL_NODE_BINARY" --print "require('path').dirname(require.resolve('@sentry/cli/package.json'))")
 [ -z "$SENTRY_CLI_EXECUTABLE" ] && SENTRY_CLI_EXECUTABLE="${SENTRY_CLI_PACKAGE_PATH}/bin/sentry-cli"
 
+[ -z "$SENTRY_FORCE_FOREGROUND"] && SENTRY_FORCE_FOREGROUND=true
```

Shellcheck (0.10.0)

[error] Couldn't parse this test expression. Fix to allow more checks.

(SC1073)